### PR TITLE
MAT-4173 Measure Highlighting and few more updates

### PR DESCRIPTION
**Pull request by praveenavas**
MAT-4173 Measure Highlighting and few more updates

### CODE REVIEW

File: cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+        //Navigate to Test Cases page and add Test Case details
+        cy.get(EditMeasurePage.testCasesTab).click()
+        cy.get(TestCasesPage.newTestCaseButton).should('be.visible')
+        cy.get(TestCasesPage.newTestCaseButton).should('be.enabled')
+        cy.get(TestCasesPage.newTestCaseButton).click()
+
+        cy.get(TestCasesPage.testCasePopulationList).should('be.visible')
+
+        cy.wait(2000)
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts:61]: do we need this wait?

File: cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        let randValue = (Math.floor((Math.random() * 1000) + 1))
+        let newCqlLibraryName = CqlLibraryName + randValue
+
+        Utilities.deleteMeasure(measureName, newCqlLibraryName, measureScoringArray[0])
+
+    })
+
+    it('Execute single Test Case and verify Measure highlighting', () => {
+
+        //Add Measure Group
```
[the remark this answers is not in the document]
Comment (praveenavas) [reply cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts:50]: yes we r checking HTML results in this test

File: cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts (hunk 1 above)
Comment (praveenavas) [reply cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts:61]: Copied from a different test which has wait. May be this is added to resolve the test failures

Comment (cvasile001) [reply cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts:61]: lets see if it works without it

File: cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (cvasile001) [reply cypress/integration/WebInterface/Test Cases/MeasureHighlighting.spec.ts:50]: cool thanks

**Pull request merged**